docs/guidelines/data-visualization/success-criteria/screenreader.md

```diff
@@ -1,8 +1,172 @@
 <br>
 
-# Screen reader
+## Screen reader
 
-Data and insights should be accessible for people using screen readers.
+All data that is accessible for sighted users should also be accessible for those using screen readers. This can be achieved through either turning the chart into an image with alternative text, adding table alternatives, or adding additional screen reader information to the visual elements. In addition, screen reader users should get the same insights as sighted people.
 
-We want to make sure that all data that's accessible for sighted users is also accessible for those using screen readers, which can be achieved through either turning the chart into an image with alternative text, adding table alternatives (recommended fallback) or adding additional screen reader information to the visual elements. In addition, screen reader users should get the same insights, with the same ease, as sighted people.
+### 1. Use correct heading elements
+![Red design tagn and blue tech tag](/dataviz-tag-design-and-tech.svg)
 
+Correct use of heading levels are necessary for screen reader users to effectively navigate a page and a chart.
+
+::: details
+How to achieve this:
+
+#### Use heading tags
+Use heading tags (`<h1>`, `<h2>`, `<h3>` etc.) instead of text elements. Normally the visual styling will follow the heading levels, but this is not always necessarily.
+
+#### Heading levels should be in a hierarchical order
+Think of the headings as an outline in the table of contents of a book. Avoid skipping heading levels.
+
+<!-- Do we only want one H1 on a page? --> 
+
+See also [(WCAG) Page Structures: Headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)
+:::
+
+### 2. Communicate visually apparent features such as trends or patterns
+![Red design tagn and blue tech tag](/dataviz-tag-design-and-tech.svg)
+
+If possible, describe trends or patterns in text, so that it is available for both screan readers and people who struggle to understanding the graph. 
```

Comment:
- ...patterns in `the` text...
- `screen readers`
- struggle to `understand` the graph